internal/gatewayapi/envoyextensionpolicy.go

```diff
@@ -614,12 +614,16 @@ func (t *Translator) buildWasms(
 	policy *egv1a1.EnvoyExtensionPolicy,
 	resources *resource.Resources,
 ) ([]ir.Wasm, error) {
+	var wasmIRList []ir.Wasm
+
+	if len(policy.Spec.Wasm) == 0 {
+		return wasmIRList, nil
```

Comment:
does the caller expect `nil` or empty list ?

Reply:
nil slice wasmIRList was previosuly returned if `range policy.Spec.Wasm` was empty. So, should be safe.